src/components/ModelTableList/CloudGroup.test.tsx

```diff
@@ -72,7 +73,26 @@ describe("CloudGroup", () => {
   });
 
   it("model access buttons are present in cloud group", () => {
-    const store = mockStore(dataDump);
+    const store = mockStore(
+      rootStateFactory.build({
+        general: generalStateFactory.build({
+          config: configFactory.build({
+            controllerAPIEndpoint: "wss://jimm.jujucharms.com/api",
+          }),
+          controllerConnections: {
+            "wss://jimm.jujucharms.com/api": {
+              user: {
+                "display-name": "eggman",
+                identity: "user-eggman@external",
+                "controller-access": "",
+                "model-access": "",
+              },
+            },
+          },
+        }),
+        juju: dataDump.juju,
```

Comment:
shouldn't this be replaced with a factory as well?

Reply:
Yes! I just haven't created the juju factories yet (that's the next thing to do). There actually is a juju state factory but it includes outdated data so it's not applicable for all tests.